deeprank/features/accessibility.py

```diff
@@ -0,0 +1,85 @@
+import logging
+
+from pdb2sql import pdb2sql
+import freesasa
+
+from deeprank.features import FeatureClass
+from deeprank.operate.pdb import get_residue_contact_atom_pairs
+
+
+_log = logging.getLogger(__name__)
+
+
+def get_atoms_of_iterest(mutant, distance_cutoff):
+
+    pdb = pdb2sql(mutant.pdb_path)
+    try:
+        atoms = set([])
+        for atom1, atom2 in get_residue_contact_atom_pairs(pdb,
+                                                           mutant.chain_id,
+                                                           mutant.residue_number,
+                                                           distance_cutoff):
+
+            # Add all atoms, even from the mutant residue itself:
+            atoms.add(atom1)
+            atoms.add(atom2)
+
+        return atoms
+    finally:
+        pdb._close()
+
+
+FEATURE_NAME = "accessibility"
+
+
+def __compute_feature__(pdb_data, featgrp, featgrp_raw, mutant):
+    "computes SASA-based features"
+
+    # Let pdb2sql tell us which atoms are around the mutant residue:
+    distance_cutoff = 8.5
+    atoms_keys = set([])
+    chain_ids = set([])
+    for atom in get_atoms_of_iterest(mutant, distance_cutoff):
+        atom_key = (atom.chain_id.strip(), int(atom.residue.number), atom.name.strip())
+        atoms_keys.add(atom_key)
+        chain_ids.add(atom.chain_id)
+
```

Comment:
So is the freesasa computed for the residues 8.5A around the mutant?

Reply:
freeSASA computes the solvent accessible surface area for the entire structure. It returns the area per atom. This feature module will only store the area's for the atoms around the mutant in the HDF5 file. We could of course, also store the area of every atom. It would eventually all be mapped to a 30x30x30 feature grid that the network takes as input.

Reply:
ok great!